packages/menu/menu-item.md

```diff
@@ -138,9 +169,16 @@ In the following example, the selected `<sp-menu-item>` represents a `value` of
 </sp-action-menu>
 ```
 
+</sp-tab-panel>
+</sp-tabs>
+
 When the `value` attribute is leveraged, the selected `<sp-menu-item>` represents a `value` of `short-key` for the `<sp-action-menu>` element.
```

Comment:
```suggestion
When the `value` attribute is leveraged, the selected `<sp-menu-item>` represents a `value` of `"short-key"` for the `<sp-action-menu>` element.
```

Reply:
@najikahalsema do we want both the backticks and the quotes?

Reply:
My thought here was that when you're writing the actual code, the syntax is `value="short-key"`, right? That's why I have both. If it's actually `value=short-key`, then we wouldn't need the quotes. What do you think?